homework/homework10/types_decorator.py

```diff
@@ -0,0 +1,51 @@
+"""Types decorator.
+This program checks the type of function parameters,
+converts them if necessary and then adds them up
+"""
+
+from functools import wraps
+
+
+def typed(types):
+    """Get type of parameter from function"""
+    def decorator(func):
+        """A decorator checks the type of function parameters
+        and converts them"""
+        @wraps(func)
+        def wrapper(*args):
+            list_arg = []
+            for i in args:
+                types(i)
```

Comment:
Please replace this loop with list comprehension.

Reply:
Fixed